### ISSUE TITLE
README.md translate

### ISSUE THREAD

**Issue by lalo-uy** [saniaxxx/RectifyPlugin#2]
Is there any chance to have a english version?
Thanks in advance

**Comment by saniaxxx** [comment]
Yes of course, I will do it when I can find the time. Are you already using this plugin?

**Comment by shunemann** [comment]
hey ALEX.
i want to use this plugin but i can`t find it after installation..

**Comment by saniaxxx** [comment]
> hey ALEX.
> i want to use this plugin but i can`t find it after installation..

Hi. You may install it by clone this repository to 'plugins' folder.
The plugin was included in the list of supported plugins and you may install in from GUI

**Comment by Alneyt** [comment]
> Yes of course, I will do it when I can find the time. Are you already using this plugin?

Hi, are you planning to create a new one compatible with craftbeerpi4 version?
Thanks and

**Comment by shunemann** [comment]
Добрый .
Я в этоге ничего не переделывал и новое не писал.
Выбрал плагины которые подходят, настроил их и все работает.

вс, 20 нояб. 2022 г., 22:07 Alneyt ***@***.***>:
